src/components/ReportCard.js

```diff
@@ -70,69 +94,59 @@ const StyledPill = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
-  width: 69px;
-  height: 28px;
+  width: 70px;
```

Comment:
Let it be whatever width and height it wants, this might well be variable content so the text could change if it was a tag or something in the backend, but even if it was hardcoded if we wanted to bump the font size up or the client asked for this to be 'Our Reports' instead the set values would restrict.

Instead drop the height and width and use padding instead to create the size going off the mock, then if it needs to grow it can.